packages/material-ui/package.json

```diff
@@ -64,7 +64,7 @@
     "webpack": "^5.53.0"
   },
   "dependencies": {
-    "@interactors/core": "^1.0.0-rc1.0",
-    "@interactors/html": "^1.0.0-rc1.0"
+    "@interactors/core": "1.0.0-rc1.0",
+    "@interactors/html": "1.0.0-rc1.0"
```

Comment:
Why does `material-ui` need core? Also, I think since material-ui is an "add-on" package, it should be a ^

Reply:
material-ui re-exports html and core stuff. And we heavly use html/core stuff in material-ui